.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 **Fixes**
 <!-- If PR only partly solves the issue, replace 'Fixes' below with 'Partially addresses' -->
-Fixes #XX by @YY
+Fixes #XX by @YourUserName
```

Comment:
I would just leave the `@` out altogether and update the instructions above to include the `@` sign. Also, the username should be of the person who created the issue, not the person who is creating the PR.

Reply:
can i change it to  this @UserName

Reply:
thank @kgodey

Reply:
I think @username would be fine.

Reply:
i have done @kgodey